src/main/java/com/cleanroommc/groovyscript/compat/mods/actuallyadditions/AtomicReconstructor.java

```diff
@@ -117,6 +117,12 @@ public RecipeBuilder type(Lens type) {
             return this;
         }
 
+        @Override
+        protected int getMaxInput() {
+            // Ignores input stack size
```

Comment:
i dont see the need for this comment in every instance where this is true - that should be implicit.

Reply:
I added this because different mods/machines cause different behaviors. I could see the following solutions:
A) do nothing.
B) do nothing unless the comment is "Ignores input stack size", in which case remove it.
C) remove it every time.
I think B) is the best, although I could see C) too.